prow/getting_started.md

```diff
@@ -23,6 +23,15 @@ gcloud container --project "${PROJECT}" clusters create prow \
   --zone "${ZONE}" --machine-type n1-standard-4 --num-nodes 2
 ```
 
+## Create cluster role bindings
+As of 1.8 Kubernetes uses [Role-Based Access Control (“RBAC”)](https://kubernetes.io/docs/admin/authorization/rbac/) 
+to drive authorization decisions, allowing admins to dynamically configure policies. 
+To create cluster resources you need to grant a user admin role in all namespaces for the cluster.
+
+```sh
+kubectl create clusterrolebinding cluster-admin-binding-"${USER}" --clusterrole=cluster-admin --user="${USER}"
```

Comment:
Personally, I believe giving away cluster-admin is not a good suggestion and prow actually needs much fewer permissions in order to work.

Reply:
For installing prow, you should need cluster-admin when you need to create the CRD though and I think that's what you want to capture here so ignore my previous comment in that case.

Reply:
yeah, my understanding is that "To create cluster resources" mostly refers to creating the CRD, perhaps that could be clarified. We'd also like to switch over to proper roles/rules though: https://github.com/kubernetes/test-infra/pull/8088#discussion_r189668346